packages/deno/index.ts

```diff
@@ -1,8 +1,2 @@
-export * from "./get-graphql-parameters.ts";
-export * from "./process-request.ts";
+export * from "@graphql-helix/core DENOIFY: DEPENDENCY UNMET (BUILTIN)";
```

Comment:
This warning seems dangerous? `DENOIFY: DEPENDENCY UNMET (BUILTIN)`

I am actually not really aware of all the deno shenanigans, maybe it is worth adding a deno integration test suite in a separate PR first, so we can verify this PR does not break the behavior?